packages/core/stories/all.stories.ts

```diff
@@ -68,6 +68,10 @@ introStories.add("Welcome", () => {
 
 				Vue
 			</a>
+
+			<a class="bx--btn bx--btn--primary last" href="https://carbon-design-system.github.io/carbon-charts/svelte" target="_blank">
```

Comment:
we really need a better UI for this section. now with the Svelte button we're definitely showing a long range of buttons